parser-typechecker/src/Unison/Util/TQueue.hs

```diff
@@ -8,7 +8,7 @@ import UnliftIO.STM hiding (TQueue)
 
 data TQueue a = TQueue (TVar (Seq a)) (TVar Word64)
 
-newIO :: (MonadIO m) => m (TQueue a)
+newIO :: forall a m. (MonadIO m) => m (TQueue a)
```

Comment:
Why flip the order?

Reply:
It's nice to be able to annotate what type of stream it is for readability, e.g. `newIO @Hash32` or whatever :)

Reply:
Ok